addons/website/static/src/snippets/s_image_gallery/000.js

```diff
@@ -24,13 +32,21 @@ const GalleryWidget = publicWidget.Widget.extend({
      * @param {Event} ev
      */
     _onClickImg: function (ev) {
-        if (this.$modal || ev.currentTarget.matches("a > img")) {
+        const clickedEl = ev.currentTarget;
+        if (this.$modal || clickedEl.matches("a > img")) {
             return;
         }
         var self = this;
-        var $cur = $(ev.currentTarget);
 
-        var $images = $cur.closest('.s_image_gallery').find('img');
+        const imageGalleryEl = clickedEl.closest(".s_image_gallery");
+        let imageEls = imageGalleryEl.querySelectorAll("img");
+        const currentImageEl = (clickedEl.matches("img") === true) ? clickedEl : clickedEl.closest("img");
```

Comment:
Line too long.

`(clickedEl.matches("img") === true)` => `clickedEl.matches("img")`
But in fact `.closest()` return the closest ancestor **or itself**, which matches the selectors. If there are no such element, null.